ROTSC/iocBoot/iocROTSC-IOC-01/config.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<ioc_config xmlns:xi="http://www.w3.org/2001/XInclude">
+<config_part>
+<ioc_desc>The HRPD/GEM/POLARIS sample changer</ioc_desc>
+<ioc_details>The HRPD/GEM/POLARIS sample changer</ioc_details>
+<macros>
+<macro name="PORT" pattern="^COM[0-9]+$" description="Serial COM Port" />
+<macro name="BAUD" pattern="^[0-9]+$" description="Serial communication baud rate, defaults to 57600." />
+<macro name="BITS" pattern="^[0-9]$" description="Serial communication number of bits, defaults to 8." />
+<macro name="PARITY" pattern="^(even)|(odd)|(none)$" description="Serial communication parity, defaults to none." />
+<macro name="STOP" pattern="^[0-9]$" description="Serial communication stop bit, defaults to 1." />
```

Comment:
Standard pattern is `(Default: number)` in brackets rather than `defaults to...`